docs/source/datasets/index.rst

```diff
@@ -183,6 +186,8 @@ homepage <https://www.fns.usda.gov/wic>`_.
 Pseudopeople can generate a simulated version of the administrative data that would be recorded by WIC. This is a yearly file of information about all 
 simulants enrolled in the program as of the end of that year.
 
+To find out more about how to generate a simulation of WIC using Pseudopeople, see :func:`pseudopeople.interface.generate_women_infants_and_children`. 
```

Comment:
I feel like we could make these links less wordy, e.g. maybe just say "API Reference: <link>" or something. I don't know if that's the best idea though.

I think it might also be good to put them right under the section titles or something, so they're easier to find. But again, not sure.

Reply:
I like having the function name here so that it's clear that it will link to the corresponding function specifically! I associate "API reference" with the collective set of functions available in a package.

Reply:
@zmbc Sorry, that was unclear. By _link_, I meant the function name like we currently have, which links to the API Reference. What I was trying to say was that I don't think we need a long sentence in _front_ of the link -- maybe we could even just put nothing in front of it, .i.e.,
```
:func:`pseudopeople.interface.generate_women_infants_and_children`
```
Or if we want to label it, then
```
API Reference: :func:`pseudopeople.interface.generate_women_infants_and_children`
```
or put something else simple in front of it.

Reply:
Or maybe:
```
Generate WIC data with :func:`pseudopeople.interface.generate_women_infants_and_children`
```

Reply:
I like that best!